### PR TITLE
Feature/50 editor config

### PR DESCRIPTION
**Pull request by NDMyroshnychenko**
Add .editorconfig file to frontend and backend projects. Use spaces for tabulation and LF as line endings

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# https://editorconfig.org
+
+root = true
+
+[*]
+charset = utf-8
+indent_style = tab
```
Comment (klesogor) [inline .editorconfig:7]: Spaces are preferable

File: slidez-core/config/checkstyle/checkstyle.xml [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+		<module name="MemberNameCheck"/>
+		<module name="MethodNameCheck"/>
+		<module name="PackageNameCheck"/>
+		<module name="ParameterNameCheck"/>
+		<module name="StaticVariableNameCheck"/>
+		<module name="TypeNameCheck"/>
+
+		<!-- Regexp -->
+		<module name="com.puppycrawl.tools.checkstyle.checks.regexp.RegexpSinglelineJavaCheck">
+			<property name="format" value="^\t* +\t*\S"/>
```
Comment (klesogor) [inline slidez-core/config/checkstyle/checkstyle.xml:116]: Use spaces instead of tabs

**Review by klesogor — CHANGES_REQUESTED** [review]

File: .editorconfig (hunk 1 above)
Comment (NDMyroshnychenko) [reply .editorconfig]: Changed indent_style **_tab_** to **_space_**

File: slidez-core/config/checkstyle/checkstyle.xml (hunk 2 above)
Comment (NDMyroshnychenko) [reply slidez-core/config/checkstyle/checkstyle.xml]: changed _**tab**_ to _**space**_

**Review by klesogor — APPROVED** [review]

**Review by VL4DYSL4V — APPROVED** [review]

**Pull request merged**